### PR TITLE
US #55: Custom fields

### PR DESCRIPTION
**Pull request by bameda**
- [x] [BACK] Custom attribnutes API endpopint.
- [x] [BACK] Import/Export custom attributes.
- [x] [BACK] Custom attributes values in US/Task/Issue models API endpoints.
- [x] [BACK] Clean custom atributes values when a custom attribute is removed.
- [x] [BACK] Custom attributes in the history.
- [x] [BACK] Import/Export custom atttributes values.
- [x] [BACK] Permissions tests.
- [x] [BACK] Custom attributes in the webhooks.
- [X] [BACK] Custom attributes in Slack integration.
- [X] [BACK] Custom attributes in Hall integration.
- [X] [BACK] Custom attributes in HipChat integration.

Related PRs:
- front: https://github.com/taigaio/taiga-front/pull/361
- doc: https://github.com/taigaio/taiga-doc/pull/46
- slack: https://github.com/taigaio/taiga-contrib-slack/pull/7
- hall: https://github.com/taigaio/taiga-contrib-hall/pull/1
- hipchat: https://github.com/taigaio/taiga-contrib-hipchat/pull/4

### CODE REVIEW

**Comment by superalex** [comment]
I looks great!, I've just added some comments for three typos.

I haven't seen anything related to import/export, are you including it in this PR too?

I think we could wait until we have the front PR ready for merging them at the same time.

**Comment by bameda** [comment]
The PR isn't ready yet.
- [x] Custom attribnutes API endpopint.
- [ ] Custom attributes values in US/Task/Issue models.
- [ ] Custom attributes in the history.
- [ ] Custom attributes in the webhooks.
- [ ] Custom attributes in Slack integration
- [ ] Import/Export custom attributes.

**Comment by superalex** [comment]
okidoki! :dancers:

File: taiga/export_import/service.py [hunk 1]
```diff
@@ -276,61 +334,68 @@ def store_role_point(project, us, role_point):
...
-        if key != 'role_points':
-            userstory_data[key] = value
-    serialized_us = serializers.UserStoryExportSerializer(data=userstory_data, context={"project": project})
-    if serialized_us.is_valid():
-        serialized_us.object.project = project
-        if serialized_us.object.owner is None:
-            serialized_us.object.owner = serialized_us.object.project.owner
-        serialized_us.object._importing = True
-        serialized_us.object._not_notify = True
+    data_data = {key: value for key, value in data.items() if key not in ["role_points", "custom_attributes_values"]}
```
Comment (superalex) [inline taiga/export_import/service.py]: It sounds like a bad name for a variable for me :)

Comment (bameda) [inline taiga/export_import/service.py]: Change in 3fa4acb

**Comment by superalex** [comment]
Great! :+1:

**Pull request merged**